src/Controllers/cli_controller.py

```diff
@@ -4,25 +4,16 @@
 from prompt_toolkit import prompt
 from Views.cli_view import CLI_View
 
-
-
 class CLI_Controller:
 
     def __init__(self):
         self._view = CLI_View()
     
-    
-    def update(self, parsed_input:list):
```

Comment:
so... removing this bricks the CLI.

Reply:
the idea was for both the gui and cli controllers to have an update method that equates to doing everything needed for a single command, then the main controller would just call update in a loop and everything else would happen regardless of which controller it possesses.

Reply:
Both controllers will do the same in update(), so I just put it back in UML_Controller. And I found it was redundant to make a separate method for the functionality, so I just put it back in run().

Reply:
Yeah, this is a quick and probably temporary fix. For now, it's working. But we could add update() function back when we have different update functionality in CLI and GUI.